arsen.musheghyan/repo/Lesson_3/exercise_11/exercise_11.py

```diff
@@ -0,0 +1,5 @@
+length = int(input("Please, enter length of a farmer’s field in feet: "))
+width = int(input("Please, enter width of a farmer’s field in feet: "))
```

Comment:
LW: If the user inputs an integer program will print an error message (ValueError). Change code to avoid this type of unwanted errors.